io/serialport/README.md

```diff
@@ -55,3 +55,22 @@ This node behaves as a tightly coupled combination of serial in and serial out n
 Send the request message in `msg.payload` as you would do with a serial out node. The message will be forwarded to the serial port following a strict FIFO (First In, First Out) queue, waiting for a single response before transmitting the next request. Once a response is received (with the same logic of a serial in node), or after a timeout occurs, a message is produced on the output, with msg.payload containing the received response (or missing in case if timeout), msg.status containing relevant info, and all other fields preserved.
 
 For consistency with the serial in node, msg.port is also set to the name of the port selected.
+
+### Port Select
```

Comment:
This info (plus any changes ;-) needs to be copied to the sidebar info tab as well. locales/en-US/25-serial.json